loadTermGraph.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
copyright header
location needs to change.

build/term/ would be my preference

Reply:
Added copyright and moved to fhir-install/src/main/docker/ibm-fhir-term-graph-loader

renamed to run.sh to match fhir-schematool pattern.